packages/block-library/src/image/image.js

```diff
@@ -597,7 +598,11 @@ export default function Image( {
 						onSelect={ onSelectImage }
 						onSelectURL={ onSelectURL }
 						onError={ onUploadError }
-					/>
+					>
+						<MenuItem onClick={ () => onSelectImage( false ) }>
```

Comment:
```suggestion
						<MenuItem onClick={ () => onSelectImage( undefined ) }>
```

Let's pass `undefined` here since `undefined|Object` are the types  `onSelectImage` method expects.